tests/phpunit/compat/class-wc-stripe-subscription-renewal-test.php

```
@@ -782,4 +782,83 @@ public function test_payment_intent_returns_non_retryable_error() {
+				],
+				'The card was declined. %s',
+			],
+			'retryable error with retries exhausted' => [
+				[
+					'type'    => 'api_error',
+					'message' => 'Mock API error',
+				],
+				'Sorry, we are unable to process the payment at this time. Reason: Mock API error %s',
+			],
+		];
+	}
```

PHPMD (2.15.0)

[error] The method provide_test_failed_renewal_note_links_request_log_url_in_new_tab is not named in camelCase. (undefined)

(CamelCaseMethodName)